src/View/User/stage.html.twig

```diff
@@ -3,5 +3,6 @@
     <link rel="stylesheet" href="/assets/css/stage.css">
 {% endblock %}
 {% block content %}
+    {% include "User/Includes/_stage_white_block.html.twig" %}
     {% include "User/Includes/_stage_grey_block.html.twig" %}
```

Comment:
use siingle quotes to be coherent throughout your project

Reply:
done